.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 npm-debug.log
 yarn-error.log
 coverage
-.idea
\ No newline at end of file
+.idea
```

Comment:
Hello, I think something like that should be put into your private gitconfig at global level.

Reply:
Ok, I'll remove it. It can indeed be a personal preference. I'm just used to do it at project level to avoid teammates from pushing those :)